src/bridge.ts

```diff
@@ -967,6 +967,7 @@ export class Bridge {
     public addAppServicePath(opts: {
         method: "GET"|"PUT"|"POST"|"DELETE",
         path: string,
+        checkToken?: boolean,
```

Comment:
```suggestion
        checkToken: boolean = true,
```

Given the `checkToken === false`, this defaults to `true`.
Let's make that explicit in the signature.

I'm a fan of defaulting to `false`. Have you considered `ignoreToken` as a variable name?

Reply:
Hm, maybe we should just have `unauthenticated` to be properly explicit. `checkToken` was used as the legacy variable name, but since that was removed in 5.0.0 and we have typings, I don't think we need to follow it.

Reply:
Oh I should have mentioned that I couldn't do a explicit signature value because it's an opts object type. I have however been explicit in the code below.

On the naming, on the whole I think that having a positive variable name like `authenticate` is probably a better look than `ignoreToken` because negative variable names are usually bad. I do concede that having default-to-true variables are a bit spooky, I hope people read the docstring :)